packages/loader/src/gltf/extensions/EXT_meshopt_compression.ts

```diff
@@ -0,0 +1,30 @@
+import { registerGLTFExtension } from "../parser/GLTFParser";
+import { GLTFParserContext, GLTFParserType } from "../parser/GLTFParserContext";
+import { GLTFExtensionMode, GLTFExtensionParser } from "./GLTFExtensionParser";
+import { MeshoptDecoder } from "./MeshoptDecoder";
+
+interface MeshOptSchema {
+  buffer: number;
+  byteOffset?: number;
+  byteLength: number;
+  byteStride: number;
+  mode: "ATTRIBUTES" | "TRIANGLES" | "INDICES";
+  count: number;
+  filter: "NONE" | "OCTAHEDRAL" | "QUATERNION" | "EXPONENTIAL";
+}
+
+@registerGLTFExtension("EXT_meshopt_compression", GLTFExtensionMode.CreateAndParse)
+class EXT_meshopt_compression extends GLTFExtensionParser {
+  override createAndParse(context: GLTFParserContext, schema: MeshOptSchema): Promise<Uint8Array> {
+    const { count, byteStride, mode, filter, buffer, byteLength, byteOffset } = schema;
```

Comment:
This cache has no advantage (taking or code clarity), but instead adds one more line

Reply:
done